src/btf.rs

```diff
@@ -89,6 +89,26 @@ impl Btf {
         Ok(ids)
     }
 
+    /// Find a list of BTF ids using a regex describing their name as a key.
```

Comment:
Not sure if  "as a key" is a typo or if I just fail to understand the sentence. I don't see any key indexing. Alternative: "Find a list of BTF ids whose names match a regex"